<!-- WaveSim Config Server

 -->
<launch>
    <!-- Launch the wave_sim config server -->
    <node pkg="wave_sim_config" type="wave_sim_config_server" name="wave_sim_config_server"
        respawn="true" output="screen" >
    </node>

</launch>
